<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_room</class>
 <widget class="QWidget" name="admin_room">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>541</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Добро пожаловать в кабинет администратора.</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="appointPushButton">
        <property name="text">
         <string>Назначить тесты</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addPushButton">
        <property name="text">
         <string>Добавить пользователя</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showPushButton">
        <property name="text">
         <string>Посмотреть результаты</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="addUserWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>115</y>
     <width>541</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QWidget" name="addWidget_child" native="true">
      <layout class="QFormLayout" name="addLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="setnamelabel">
           <property name="text">
            <string>Имя</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="setstatuslabel">
           <property name="text">
            <string>Статус</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="setloginlabel">
           <property name="text">
            <string>Логин</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="setpasswordlabel">
           <property name="text">
            <string>Пароль</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLineEdit" name="nameLineEdit"/>
         </item>
         <item>
          <widget class="QComboBox" name="statusComboBox">
           <item>
            <property name="text">
             <string>Администратор</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Испытуемый</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="loginLineEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="passwordLineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addUserPushButton">
      <property name="text">
       <string>Добавить пользователя</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="assignWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>115</y>
     <width>541</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QComboBox" name="userChooseComboBox"/>
    </item>
    <item>
     <widget class="QListWidget" name="testChooseWidget">
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="assignPushButton">
      <property name="text">
       <string>Назначить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="resultWidget" native="true">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>115</y>
     <width>531</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QComboBox" name="resultChooseComboBox"/>
    </item>
    <item>
     <widget class="QLabel" name="result_label">
      <property name="text">
       <string>Выберите пользователя</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
